spdockit-spo/office-365-groups-and-onedrive/office-365-management-wizard.md

```diff
@@ -0,0 +1,63 @@
+---
+title: Office 365 Management Wizards
+description: This article describes how to manage Office 365 groups and teams using the Groups and Membership Wizards. 
+author: Tomislav Sirovec
+date: 14/11/2017
+---
+
+This article describes how to use SPDocKit's Groups and Membership Office 365 Wizards.  
+Wizard buttons are visible only in __live mode__ of Permissions Explorer and when either Groups and Teams node is selected or a specific group.  
+Wizard Scope: entire tenant
+
+
+## Groups section
+
+### Delete Groups wizard  
+-will delete the selected groups/s from entire tenant.  
+
+1. By default, __Target__ is entire tenant. If you have more than one tenant connected, there is a dropdown where you can choose which one you want.
+1. On __Group Selection__ step pick the groups you want to delete.  
+    - You can pick specific groups from a dropdown
+    - Match teams and groups by name that contains
+    - Match teams and groups that match the specified regular expression
+1. __Preview step__ will show you changes before executing them. Click Next.
+1. The last step shows user the changes that have been made. If some errors have occurred they will be displayed here. It is possible to save this log to disk as a .txt file using Save Log button. In addition, SPDocKit will create Event Log entry in custom SPDocPerm event log. There will be information about changes that were made as well as who made those changes.
+
+## Membership section
+
+### Owners wizard  
+-will add or remove owners from Office 365 Teams and Groups.
+1. Select whether you want to add or remove owners from Office 365 Teams and Groups
+   - Add Owner/s to specified groups
+   - Remove owner/s from specified groups
+   - Remove owner/s from all groups
+2. In the next step define the __Target__ for your action. By default that is entire tenant. If you have more than one tenant connected, there is a dropdown where you can choose which one you want.
+3. On the __Principal Selection__ step populate the people picker with the users of interest.  
```

Comment:
Add instructions how can these users be imported - by email, name or using csv or txt file?
For every wizard and action where this is possible.